src/Aqovia.Utilities.SagaMachine/StatePersistance/RedisKeyValueStore.cs

```diff
@@ -130,53 +138,58 @@ public void Remove(string key)
         /// <returns></returns>
         public async Task<bool> TakeLock(string key, Guid lockToken, double milliseconds)
         {
-            var retryCount = 3;
+            var maxRetryAttempt = 3;
```

Comment:
If I read correctly, this is the max number of attempts, and not retries (retries would be ).

If that's true, than I'd call it something akin to `maxNumberOfAttempts`